content/docs/user-guide/how-to/undo-adding-data.md

```diff
@@ -3,41 +3,33 @@
 There are situations where you want to stop tracking data added previously.
 Follow the steps listed here to undo `dvc add`.
 
-Let's first add a data file into an example <abbr>project</abbr> using
-`dvc add`, which creates a `.dvc` file to track the data:
+Let's first add a data file into an example <abbr>project</abbr>, which creates
```

Comment:
I would eventually expect this how-to to become "Remove data" or something ... inverse of `dvc add` sounds a bit too specific?

Reply:
though we can keep creating more and more small recipes ... and link them after

Reply:
Good ideas. Should we write this in #899? We probably don't want to expand the scope of this PR (even further 😅).

Reply:
Added

- Explore changing the existing how-to "Undo Adding Data" to "Remove data" — what other ways to remove data other than un-doing `add` are there?

to the how-tos epic. Resolving here 🙂